publify_core/app/assets/stylesheets/sidebar_admin.css.scss

```diff
@@ -22,7 +22,7 @@
   padding: 45px;
   border: 1px solid #ccc;
 }
-.sidebar_item {
+.sidebar-item {
```

Comment:
Why is this necessary?

Reply:
I changed this because this controlled the font styling for the Archive sidebar. I could've left this css file alone and only changed the class on the HTML file. Now I know it is an unnecessary change.

Reply:
Yes, `sidebar-item` is not a class used anywhere. 

Please push changes addressing my comments to this same branch.